### PR TITLE
docs: add high availability article

### PR DESCRIPTION
**Pull request by mbshields**
**What type of PR is this?**

documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/articles/high-availability.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Deploying a Highly Available zot Registry
+
+> :point_right: A highly available zot registry can be easily implemented using zot's registry synchronization feature.
+
+In the zot configuration, the `sync` extension allows a zot instance to mirror another zot instance with various container image download policies, including on-demand and periodic downloads. You can use the zot `sync`  function combined with a load balancer such as [HAProxy](https://www.haproxy.com) to implement a highly available registry.
+
+Two failover configurations are possible:
+
+* Active/active
```
Comment (rchincha) [inline docs/articles/high-availability.md:9]: Would make active/active as second option.

File: docs/articles/high-availability.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+> :pencil2: The highly available zot registry described in this article differs from [zot clustering](clustering.md). Although zot clustering provides a level of high availability, the instances share common storage, whose failure would affect all instances. In the method described in this article, each instance has its own storage, providing an additional level of safety.
+
+For details of configuring the `sync` extension, see [OCI Registry Mirroring With zot](mirroring.md).
+
+## Configuring an active/active registry
+
+An active/active zot registry can be implemented between two zot instances by configuring the `sync` extension in each instance to point to the other instance.  In this scheme:
+
+* a load balancer such as HAProxy or a [DNS-based routing](https://coredns.io/plugins/loadbalance/) scheme is deployed for round-robin load balancing between zot instances
```
Comment (rchincha) [inline docs/articles/high-availability.md:25]: Furthermore, one has to manage traffic split (path based routing) so some traffic goes to one instance and rest goes to another.
https://www.haproxy.com/blog/path-based-routing-with-haproxy

File: docs/articles/high-availability.md (hunk 1 above)
Comment (mbshields) [reply docs/articles/high-availability.md:9]: Swapped order of A/S and A/A in the intro and in the configuration sections

File: docs/articles/high-availability.md (hunk 2 above)
Comment (mbshields) [reply docs/articles/high-availability.md:25]: Is this to keep a repo from being fragmented across two instances?

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**